Dockerfile

```diff
@@ -1,21 +1,30 @@
-FROM archlinux
+FROM ubuntu:20.04
 
-
-RUN pacman --noconfirm -Sy qrencode \
-                           zbar \
-                           imagemagick \
-                           git \
-                           ttf-roboto \
-                           fontconfig \
-                           diffutils
-
-ENV FONT=Roboto-Regular \
+ENV DEBIAN_FRONTEND=noninteractive \
+    FONT=roboto \
     OUTPUT_DIR=/target/ \
     INPUT_DIR=/target/
 
+
+RUN apt-get update && apt-get install \
+    --no-install-recommends -y \
+    ca-certificates \
+    imagemagick \
+    fonts-roboto \
+    libzbar-dev \
+    qrencode \
+    wget && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+
+RUN wget -q https://linuxtv.org/downloads/zbar/binaries/zbar-0.23.91-ubuntu-20.04.tar.gz
```

Comment:
What do you think if we combine this layer with the layer in line 22 to reduce layers?